src/graphic/minimap_renderer.cc

```diff
@@ -160,15 +162,16 @@ void do_draw_minimap(Texture* texture,
 	const int32_t mapwidth = map.get_width();
 
 	for (uint32_t y = 0; y < surface_h; ++y) {
-		Widelands::Coords coords(
-		   Widelands::Coords(top_left.x, top_left.y + ((layers & MiniMapLayer::Zoom2) ? y / 2 : y)));
-		map.normalize_coords(coords);
-		Widelands::FCoords f = map.get_fcoords(coords);
-		Widelands::MapIndex i = Widelands::Map::get_index(f, mapwidth);
 		for (uint32_t x = 0; x < surface_w; ++x) {
-			if (x % 2 || !(layers & MiniMapLayer::Zoom2)) {
```

Comment:
Disclaimer: I have no idea how this function `do_draw_minimap` is working. But: This if() clause doesn't appear in your version. Is this correct? If you say so, that is good enough for me. (Wild guess: The if() makes sure that no useless pixels are drawn by small zoom?)

Reply:
That got me my had scratching as well, for a while. It only fills every second pixel when zooming. This cannot be used anymore, since we have a variable zoom factor now. This would also result in a vertical interlaced picture.